src/main/java/com/example/gradecalculator/model/UserEditTO.java

```diff
@@ -0,0 +1,17 @@
+package com.example.gradecalculator.model;
+
+public class UserEditTO {
+    private String firstName;
+    private String lastName;
```

Comment:
There are validation attributes in the corresponding controller post method, you should use attribute validation for at least required input (@NotBlank)

Reply:
btw: i found out that if you write smth with @ mindlessly it will tag other github users 🤐<br>
![image](https://github.com/IoT-Ausbildung/grade-calculator/assets/130460027/8fba6e79-be29-4608-bad3-8df0750f513e)